python/cudf_polars/cudf_polars/dsl/utils/aggregations.py

```diff
@@ -158,9 +158,9 @@ def decompose_single_agg(
         is_median = agg.name == "median"
         is_quantile = agg.name == "quantile"
 
-        is_group_quantile_supported = plc.traits.is_integral(
+        is_group_quantile_supported = plc.traits.is_fixed_width(
```

Comment:
cc @Matt711 update this condition to accept decimal types xref https://github.com/rapidsai/cudf/pull/19669